renku/cli/pull.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017, 2018 - Swiss Data Science Center (SDSC)
+# A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+# Eidgenössische Technische Hochschule Zürich (ETHZ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""Pull data from LFS."""
+
+import click
+
+from ._client import pass_local_client
+
+
+@click.group()
+def pull():
+    """Pull data from LFS."""
+
+
+@pull.command()
+@click.argument('filename', type=click.Path(exists=True))
```

Comment:
```suggestion
@click.argument(
    'paths',
    type=click.Path(exists=True, dir_okay=True),
    nargs=-1,
    required=True,
)
```

The code should be updated to support multiple files and directories to work similarly to other renku commands.

Reply:
so should the command be called `path` instead of `file`?

Reply:
this is now done with the simplifications from b18724f